### PR TITLE
Rerender

### PR DESCRIPTION
**Pull request by st3107**
relase 0.8.2.

https://github.com/xpdAcq/xpdtools/releases/tag/0.8.2

### CODE REVIEW

**Comment by st3107** [comment]
Hi @mrakitin . I fixed the build. Ready for review.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,25 +18,26 @@ build:
...
+    - pims
+    - fabio
+    - numba
+    - matplotlib-base
+    - tqdm
     - scikit-learn
-    - scipy
     - tomopy
-    - tqdm
+
```
Comment (mrakitin) [inline recipe/meta.yaml:40]: We tend to sort the dependency list alphabetically for easier mainenance. Could you please sort it?

**Review by mrakitin — CHANGES_REQUESTED** [review]
Looks good, but needs to comply the formalities. Please see below.

File: recipe/meta.yaml (hunk 1 above)
Comment (st3107) [reply recipe/meta.yaml:40]: Thank you for the suggestion. It's done.

Comment (mrakitin) [reply recipe/meta.yaml:40]: Thanks for sorting the deps.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -18,14 +18,13 @@ build:
     - image_to_iq = xpdtools.cli.process_tiff:run_main
 
 requirements:
-  host:
+  build:
```
Comment (mrakitin) [inline recipe/meta.yaml]: Why was it changed to `build`? See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host for details. I am going to revert it back, and merge after successful build.
```suggestion
  host:
```

**Review by mrakitin — COMMENTED** [review]
Almost good to go. I have a concern regarding renaming the `host` section to `build`, and going to fix it myself.

**Review by mrakitin — APPROVED** [review]

**Pull request merged**